NewDawn/NewDawn/ViewControllers/ViewController.swift

```diff
@@ -144,9 +144,6 @@ extension UIViewController {
     // A helper function to make text field fancier
     func polishTextField(textField: UITextField) -> Void {
         textField.setLeftPaddingPoints(10)
-        textField.layer.cornerRadius = 20
-        textField.layer.borderWidth = 1.0
```

Comment:
No need to delete these lines to hide the borders. You can use storyboard to hide them, like using the first option in border style below. 
<img width="336" alt="Screen Shot 2019-05-04 at 8 47 57 PM" src="https://user-images.githubusercontent.com/40241750/57186461-ead90700-6ead-11e9-8f15-f76f98f9b5f2.png">

Reply:
It's dangerous to change function like this, since this could be used somewhere else.